src/examples/renaming_and_moving_objects.cpp

```diff
@@ -0,0 +1,75 @@
+#include <H5File.hpp>
+#include <H5Group.hpp>
+#include <H5DataSet.hpp>
+#include <H5DataSpace.hpp>
+#include <H5Attribute.hpp>
+
+using namespace HighFive;
+
+int main(int argc, char *argv[])
```

Comment:
```suggestion
int main(void)
```
This is why the CI is failing